gcip/core/job.py

```diff
@@ -200,16 +200,16 @@ def _extend_name(self, name: Optional[str]) -> None:
         if name:
             self._name += "-" + name.replace("_", "-")
 
-    def _extend_stage(self, stage: Optional[str]) -> None:
+    def _extend_stage_value(self, stage: Optional[str]) -> None:
```

Comment:
Why do get `stages` underscores and names minuses?

Reply:
This is just a design decision. So you can easily distinguish between stage and job name when browsing the generated pipeline yaml. Currenlty I am not sure, but I think one of them can't handle minuses or dashes.
Is that anwer fine for you?

Reply:
Yepp